Rules.md

```diff
@@ -1394,6 +1395,34 @@ Option | Description
 --- | ---
 `--trimwhitespace` | trim trailing space. "always" (default) or "nonblank-lines"
 
+## typeMarks
```

Comment:
@calda I was fiddling with this idea at one point but clearly did not get very far. The idea was to add autocorrect for https://github.com/airbnb/swift/issues/54. This tag has no expectations associated with it and is purely informational 👍

Reply:
Omg nice! This would also be a great rule to bring back in the future.

Reply:
Agreed

Reply:
nicklockwood/SwiftFormat#747 :)